about.html

```diff
@@ -0,0 +1,142 @@
+      </div>
+     <ul class="menu">
+       <li><a href=""><img class="closebtn" src="images/AppBar.png" alt=""></a></li>
+       <li><a href="index.html" class="ul-menu">About</a></li>
+       <li><a href="" class="ul-menu">program</a></li>
+       <li><a href="" class="ul-menu">News</a></li>
+       <li><a href="" class="ul-menu">Join</a></li>
+       <li><a href="" class="ul-menu">Sponsor</a></li>
+       <li><a href="" class="child-7">Compaign</a></li>
+     </ul>
+          <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+        </div>
+    </nav>
+    <div class="wrapper">
+        
+      <div class="intro">
+        
+        <h3 class="title-about">"Hello Freshers!"</h3>
+        <h1 class="about-h1">Mobile first design strategy</h1>
+        <p class="about-info">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Impedit
+          magnam odit praesentium excepturi eveniet, dolor dolorem ex incidunt
+          perferendis
+        </p>
+        <p class="caption-about">2020/30/10 Join us for better knowledge</p>
+        <p class="caption-about1">@Kabul University Afghanistan</p>
+      </div>
+      <div class="main-program">
+        
+        <h2 class="main-program1">Main program</h2>
+        <div class="main">
+        <div class="lecturesection">
+         <img src="images/program_icon_01.png" alt="">
+          <h3 class="lecture">Lecture</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur</p>
+        </div>
+        <div class="lecturesection">
+          <img src="images/program_icon_02.png" alt="">
+          <h3 class="lecture">UI Exhibition</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit</p>
+        </div>
+        <div class="lecturesection">
+          <img src="images/program_icon_03.png" alt="">
+          <h3 class="lecture">Forum</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur.</p>
+        </div>
+        <div class="lecturesection">
+         <img src="images/program_icon_04.png" alt="">
+          <h3 class="lecture">Workshop</h3>
+          <p class="lecture-p">Lorem ipsum dolor</p>
+        </div>
+        <div class="lecturesection">
+        <img src="images/program_icon_05.png" alt="">
+          <h3 class="lecture">Ignit</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur</p>
+        </div>
+        </div>
+        <a class="see" href="">See the whole program</a>
+       
+        <a class="button" href="">Get your Tickets</a>
+      </div>
+        <h2 class="feature-speaker">featured Speakers</h2>
+        <hr class="line">
+      <div class="featured">
+       
+
+        </div>   
+          
+        <div class="more">
+          <h4 class="more-speaker">more </h>
+          <a href="" class="more-image"><img  class="more-img" src="images/Disabled.svg" alt=""></a>
+        </div>
```

Comment:
**[OPTIONAL]** - I currently cannot view the images for the featured speakers, please try to fix this issue.
Your project:
![featured speakers current](https://user-images.githubusercontent.com/73607512/190499551-4c8b2db2-8d4c-4fb9-b01d-a382dc4dd7f2.PNG)